region/norfair/east.json

```diff
@@ -3276,7 +3280,9 @@
         {
           "name": "Yapping Maw",
           "quantity": 3,
-          "betweenNodes": [ 1, 2 ]
+          "betweenNodes": [ 1, 2 ],
+          "dropRequires": "never",
+          "note": "Possible but requires multiple spikehits in most configurations so rarely worth it"
```

Comment:
The drops are obtainable with Grapple, the issue is actually killing them. Not like farming Yapping Maws would ever be viable anyway.